openssl/src/ssl/mod.rs

```diff
@@ -3114,6 +3112,142 @@ impl SslRef {
         }
         Ok(())
     }
+    #[cfg(not(boringssl))]
+    pub fn set_method(&mut self, method: SslMethod) -> Result<(), ErrorStack> {
```

Comment:
It would be good to document the functions you're adding, you can mostly get away with stealing the docs from [SslContextBuilder](https://docs.rs/openssl/latest/openssl/ssl/struct.SslContextBuilder.html) although they may need a couple of changes.

Reply:
Could you document the other functions too? (and put some newlines in?)